examples/SwissPaymentSlipFpdf_Orange_Slip.php

```diff
@@ -23,7 +23,7 @@
 $time_start = microtime(true);
 
 // Make sure the classes get auto-loaded
-require __DIR__.'/../vendor/autoload.php';
+$loader = require __DIR__. '/../vendor/autoload.php';
```

Comment:
Is there any reason why you assign the loader to a variable?
I think we don't need it through this short example.